### DISCUSSION TITLE
How do I find the source of warnings?

### DISCUSSION THREAD

**Discussion by rmcastil** [LazyVim/LazyVim#709] [Q&A] [answered]
When I open up an `.html` file with Handlebar expressions. I get infinite warnings

![image](https://user-images.githubusercontent.com/155443/236028928-ccb60103-0160-42f7-9fa2-548f28617001.png)

I can't tell if this a treesitter issue or an lsp issue and quite frankly I'm not sure how to diagnose it.

**Comment by amaanq** [comment, marked as answer]
LSP, :LspLog

**Reply by rmcastil** [reply]
Thank you so much!
